docs/userguide/editing.md

```diff
@@ -27,73 +27,153 @@ rather than Google Sites.
+In the case of a single file edit (ex: typo correction), a convenient way to
+create the pull request is to follow the "Edit this page on Github" link at
+the bottom of the page and [use GitHub to edit the file](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository)
+and create the new pull request.
 
-Major chapters have Introduction pages, and further sections in a chapter are
-subpages of that main chapter page. The navigation bar is a manually edited
-sidebar accessible (if you are logged in and have edit rights) from Site
-settings/Change appearance.
-
-Page URLs should use lowercase letters and no hyphens.
+After the source branch for a potential pull request is made, the changes can 
+be previewed using the fork repository that contains the source branch.  The
+fork repository must be [configured to serve GitHub Pages](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source)
+from the branch and the Markdown pages root directory (`/docs`).  Note: there is a
+latency of several minutes before GitHub Pages renders new changes, and 
+[errors](https://docs.github.com/en/github/working-with-github-pages/about-jekyll-build-errors-for-github-pages-sites)
+prevent the changes from being rendered.
 
-See the sitemap linked from the bottom of the navigation bar.
+## Jekyll Theme Configuration
 
-Most pages have an automatic Table of contents. On a new page, after entering
-some contents, return to the very top of the page contents, select Insert/Table
-of contents, save, then change it to Right-aligned and turn on Wrap.
+Jekyll site-wide configurations are stored
+in the `_config.yml` file. The current theme being used is
+[Just the Docs](https://pmarsceill.github.io/just-the-docs/).
 
-## Common Styles
+Due to incompatibilities between the GitHub's built-in implementation of the
+CommonMark Markdown parser, the default Jekyll Markdown parser, Kramdown, is
+used.
 
-We want to use common styles for code samples, notes and such. Since Google
-Sites does not offer a site-wide CSS style sheet, please copy special items from
-here, paste and modify their text, rather than creating them from scratch.
+### Document Structure
 
-For headings, and for standard text styles like **bold**, *italic*,
-~~strike-through~~, ... please use standard headings styles from Sites.
+Major chapters have Introduction pages, and further sections in a chapter are
+subpages of that main chapter page. The navigation bar is generated from the
+information provided in the headers section of the Markdown file according to 
+the theme's [navigation annotations](https://pmarsceill.github.io/just-the-docs/docs/navigation-structure/).
 
-### Code
+### Syntax
 
-**New:** Use the Format menu styles for Code (inline) and Blockquote Code
-(multi-line).
+[Standard markdown](https://commonmark.org/) (CommonMark) should be preferred over custom
```

Comment:
Should we link to Kramdown docs instead?

Reply:
Done.